src/lib/plot/series/abstract-series.js

```diff
@@ -50,7 +50,8 @@ export default class AbstractSeries extends PureRenderComponent {
       onSeriesMouseOut: React.PropTypes.func,
       onSeriesClick: React.PropTypes.func,
       onNearestX: React.PropTypes.func,
-      animation: AnimationPropType
+      animation: AnimationPropType,
+      title: React.PropTypes.string
```

Comment:
How about calling this `label` since they're referred to as `seriesLabels` in other parts of the code? Alternatively, change `seriesLabels` to seriesTitles`, as long as there's consistent terminology.